tests/integration/test_optislang/test_optislang_integration.py

```diff
@@ -1,13 +1,16 @@
+from collections import OrderedDict
 import os
 from pathlib import Path
 import shutil
+import tempfile
 
 import pytest
 from util.meshing_workflow import mixing_elbow_geometry  # noqa: F401
 
 import ansys.fluent.core as pyfluent
 from ansys.fluent.core import examples
-from ansys.fluent.core.utils.fluent_version import FluentVersion
+from ansys.fluent.core.filereader.case_file import CaseFile
+from ansys.fluent.core.filereader.casereader import CaseReader
```

Comment:
@cj-hodgson `CaseReader` is just an alias for `CaseFile` for backward compatibility. You can use `CaseFIle` everywhere you would use `CaseReader`.